app/controllers/application_controller.rb

```diff
@@ -2,6 +2,25 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_locale
 
+  respond_to :html, :js
+
+  def index
+  end
+
+  def invite_friends
+    invite_friends = params.require(:invite_friends).permit(:emails)
+
+    emails = invite_friends[:emails]
+    emails = emails.gsub(' ', '').split(',')
+
+    if @invite_friends_state = FriendsService.invite(emails)
+      flash.now[:notice] = 'Ваши друзья успешно приглашены на сайт.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.